azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/reducer/CallingReducer.kt

```diff
@@ -16,7 +16,10 @@ internal class CallStateReducerImpl : CallStateReducer {
     override fun reduce(callingState: CallingState, action: Action): CallingState {
         return when (action) {
             is CallingAction.StateUpdated -> {
-                callingState.copy(callingStatus = action.callingState, joinCallIsRequested = false)
+                callingState.copy(
+                    callingStatus = action.callingState, joinCallIsRequested = false,
```

Comment:
I know not part of your code, however

joinCallIsRequested = false 
is very strange in this reducer. It feels like it should be a part of another action, i.e. updateJoinCallIsRequested(true/false).

Reply:
As per discussion we will fix for GA.